lib/LaTeXML/Util/Image.pm

```diff
@@ -290,6 +303,7 @@ sub image_graphicx_complex {
 
   # # Wastefully preread the image to get it's initial size
   my ($w0, $h0) = image_size($source);
+  return unless $w0 && $h0;
```

Comment:
Actually, @brucemiller , if you re handling this case directly here, would you be interested in seeing if my earlier PR doesn't do an equally fine job at this?

See: https://github.com/brucemiller/LaTeXML/pull/1786